scripts/plots/cam/exevs_cam_headline_plots.sh

```diff
@@ -90,7 +90,10 @@ fi
    find ${DATA}/${VERIF_CASE}/* -name "*.png" -type f -print | tar -cvf ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar --transform='s#.*/##' -T -
 
 if [ $SENDCOM = YES ]; then
-    cpreq -v ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar ${COMOUTplots}/.
+    FILE=${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar
+    if [ -s "$FILE" ]; then
+       cp -v ${VDATE} ${COMOUTplots}/.
```

Comment:
I think ${VDATE} should be ${FILE}.

Reply:
@MarcelCaron-NOAA - can we ensure that this line is correct?